<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-5M7AM2E" failures="0" tests="16" name="com.zaigo.reactui.CreateUserTestCases" time="128.402" errors="1" timestamp="2022-09-15T10:55:45 IST" skipped="0">
  <testcase classname="com.zaigo.reactui.CreateUserTestCases" name="loginPage" time="3.332"/>
  <system-out/>
  <testcase classname="com.zaigo.reactui.CreateUserTestCases" name="modulePage" time="20.745"/>
  <system-out/>
  <testcase classname="com.zaigo.reactui.CreateUserTestCases" name="reqFieldUser" time="4.766"/>
  <system-out/>
  <testcase classname="com.zaigo.reactui.CreateUserTestCases" name="conReqField" time="2.854"/>
  <system-out/>
  <testcase classname="com.zaigo.reactui.CreateUserTestCases" name="reqOrganization" time="4.817"/>
  <system-out/>
  <testcase classname="com.zaigo.reactui.CreateUserTestCases" name="manditoryField" time="6.988"/>
  <system-out/>
  <testcase classname="com.zaigo.reactui.CreateUserTestCases" name="inputUserLocationFieldReq" time="7.092"/>
  <system-out/>
  <testcase classname="com.zaigo.reactui.CreateUserTestCases" name="conManditoryField" time="0.507"/>
  <system-out/>
  <testcase classname="com.zaigo.reactui.CreateUserTestCases" name="inputContractorLocationFieldReq" time="6.736"/>
  <system-out/>
  <testcase classname="com.zaigo.reactui.CreateUserTestCases" name="UserModule" time="15.479"/>
  <system-out/>
  <testcase classname="com.zaigo.reactui.CreateUserTestCases" name="listValidate" time="2.150"/>
  <system-out/>
  <testcase classname="com.zaigo.reactui.CreateUserTestCases" name="searchValidate" time="6.936"/>
  <system-out/>
  <testcase classname="com.zaigo.reactui.CreateUserTestCases" name="invalidSearch" time="6.586"/>
  <system-out/>
  <testcase classname="com.zaigo.reactui.CreateUserTestCases" name="conDatas" time="12.274"/>
  <system-out/>
  <testcase classname="com.zaigo.reactui.CreateUserTestCases" name="alreadyExistData" time="6.555"/>
  <system-out/>
  <testcase classname="com.zaigo.reactui.CreateUserTestCases" name="contractExistingData" time="20.585">
    <error message="Expected condition failed: waiting for visibility of element located by By.xpath: //span[text()=&#039;User have been deleted successfully&#039;] (tried for 10 second(s) with 500 milliseconds interval)" type="org.openqa.selenium.TimeoutException">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for visibility of element located by By.xpath: //span[text()='User have been deleted successfully'] (tried for 10 second(s) with 500 milliseconds interval)
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:95)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:272)
at com.zaigo.pageobjects.CreateUserPage.verifyDeleteMessage(CreateUserPage.java:999)
at com.zaigo.pageobjects.CreateUserPage.deleteField(CreateUserPage.java:1027)
at com.zaigo.reactui.CreateUserTestCases.contractExistingData(CreateUserTestCases.java:154)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//span[text()='User have been deleted successfully']"}
  (Session info: chrome=105.0.5195.102)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-5M7AM2E', ip: '192.168.0.239', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.3'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 105.0.5195.102, chrome: {chromedriverVersion: 105.0.5195.52 (412c95e51883..., userDataDir: C:\Users\ZAIGOP~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:59741}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: e5b86139b0d2e37a10f4fc8e7d475fa5
*** Element info: {Using=xpath, value=//span[text()='User have been deleted successfully']}
at jdk.internal.reflect.GeneratedConstructorAccessor12.newInstance(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:428)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:205)
at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:201)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:249)
... 31 more
]]>
    </error>
  </testcase> <!-- contractExistingData -->
  <system-out/>
</testsuite> <!-- com.zaigo.reactui.CreateUserTestCases -->
